ram/hram.asm

```diff
@@ -330,8 +330,7 @@ hGymTrashCanRandNumMask::
 	db
 
 NEXTU
-hFFDB:: db
-hFFDC:: db
+hInteractedWithBookshelf:: db ; set to 0 if you interacted with a bookshelf, $FF if not when attempting to interact with a hidden object
```

Comment:
```suggestion
; when attempting to interact with a hidden object,
; set to 0 if you interacted with a bookshelf, $FF if not
hInteractedWithBookshelf:: db
```

Reply:
Also since `hFFDC` is gone now, this can merge with the union above it (and I think we don't need the comment after all):
```
NEXTU
hItemCounter::
hSavedCoordIndex::
hMissableObjectIndex::
hInteractedWithBookshelf::
hGymTrashCanRandNumMask::
	db
ENDU
```